### ISSUE TITLE
Vertical Scroll for Mobile View

### ISSUE THREAD

**Issue by Lijun-Yu** [scs-master-deans-committee/scs-master-deans-committee.github.io#4]
For the mobile view, we may consider using a vertical scroll layout instead of the current horizontal scroll one.

In fact, if we want to add several words as an abstract for each meeting, the vertical one may be more suitable. Just a thought.

**Comment by anant-kaushik** [comment]
Yeah the mobile website is not mobile responsive yet. Vertical view is a great idea.
